contracts/CourtRoom.sol

```diff
@@ -98,6 +129,9 @@ contract SwearGame is Owned {
 
 	}
 
+
+
+
 	function deposit(uint256 _depositAmount) onlyOwner payable public returns(bool){
 
 		require(sampleToken.balanceOf(owner) >= _depositAmount);
```

Comment:
@zelig can you please elaborate more on that?
a registration of a client to the game (which should be done only by the service) check if there is enough deposits in the contract.